<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-D36JS0L" ignored="0" name="Class Test Execution" tests="4" failures="1" timestamp="12 Sep 2018 13:13:21 GMT" time="0.085" errors="0">
  <testcase name="preconditions" time="0.0" classname="testNGDemo.TestSkipDemo"/>
  <testcase name="tearDown" time="0.0" classname="testNGDemo.TestSkipDemo"/>
  <testcase name="testCode" time="0.0" classname="testNGDemo.TestSkipDemo">
    <failure type="java.lang.AssertionError" message="expected [10] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [5]
at testNGDemo.TestSkipDemo.testCode(TestSkipDemo.java:27)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testCode -->
  <testcase name="testInAnotherClass" time="0.0" classname="testNGDemo.TestSkipDemo2">
    <skipped/>
  </testcase> <!-- testInAnotherClass -->
</testsuite> <!-- Class Test Execution -->
